backends/dnf/pk-backend-dnf.c

```diff
@@ -3787,7 +3840,9 @@ pk_backend_get_update_detail_thread (PkBackendJob *job, GVariant *params, gpoint
 					      NULL /* updated */);
 
 		g_ptr_array_unref (references);
+		#ifndef HAVE_HY_QUERY_GET_ADVISORY_PKGS
```

Comment:
it's more usual to have the #ifdef without an indent

Reply:
done